helper.js

```diff
@@ -149,8 +149,8 @@ function attributeEntry(el, label, value) {
 
 function createIframe(opt, width, height) {
   el = document.createElement("iframe")
-  width = typeof width !== 'undefined' ? width : '525px';
-  height = typeof height !== 'undefined' ? height : '350px';
+  width = typeof width !== 'undefined' ? width : '525px'
```

Comment:
Go ahead

On June 8, 2016 11:29:20 PM GMT+02:00, eberhab <notifications@github.com> wrote:
>> @@ -149,8 +149,8 @@ function attributeEntry(el, label, value) {
>>  
>>  function createIframe(opt, width, height) {
>>    el = document.createElement("iframe")
>> -  width = typeof width !== 'undefined' ? width : '525px';
>> -  height = typeof height !== 'undefined' ? height : '350px';
>> +  width = typeof width !== 'undefined' ? width : '525px'
>
>can we set this to 100% instead of 525px? I just tested it, looks the
>same but should be more flexible in case on some devices the sidebar is
>narrower.
>
>---
>You are receiving this because you modified the open/close state.
>Reply to this email directly or view it on GitHub:
>https://github.com/plumpudding/hopglass/pull/38/files/85999d9e4115d91313b1783d25a9105cbeed2d2f#r66343773

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.